pychiquito/tests.py

```diff
@@ -1,27 +1,32 @@
 from chiquito_ast import (
-    StepType,
+    ASTStepType,
     ASTConstraint,
     TransitionConstraint,
     InternalSignal,
     ForwardSignal,
     SharedSignal,
     FixedSignal,
 )
-from query import Internal, Forward, Shared, Fixed
+from query import Internal, Forward, Shared, Fixed, StepTypeNext
 from expr import Const, Sum, Mul
 
-########
-# test #
-########
-# print(Internal(InternalSignal("a")).__json__())
-# print(Forward(ForwardSignal(1, "a"), True).__json__())
-# print(Shared(SharedSignal(0, "a"), 2).__json__())
-# print(Fixed(FixedSignal("a"), 2).__json__())
-# print(StepTypeNext(StepType.new("fibo")).__json__())
-# print(ASTConstraint("constraint", Sum([Const(1), Mul([Internal(InternalSignal("a")), Const(3)])])).__json__())
-# print(TransitionConstraint("trans", Sum([Const(1), Mul([Internal(InternalSignal("a")), Const(3)])])).__json__())
+print(Internal(InternalSignal("a")).__json__())
```

Comment:
A test is not printing, you would have to do assertions or removing this

Reply:
Deleted as UUID change each time so it's hard to do assertions